### DISCUSSION TITLE
【求助】怎么发图片

### DISCUSSION THREAD

**Discussion by zzz-Rope** [share121/inter-knot#2781] [其他]
为什么我选择的图片发上来是链接？<img src="https://img-static.mihoyo.com/communityweb/upload/3a2b2e0860560ccc7f6556ef547f12c8.png" alt="若陀龙王-挠头" width="75" >

**Comment by isle200inf** [comment]
图片拖到输入框就行，链接不用管，或者你可以了解一下Markdown基本语法[**官方**](https://markdown.com.cn/basic-syntax/)，[**github**](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

**Reply by isle200inf** [reply]
拖上来应该是这样的`![文件名](链接)`

**Reply by zzz-Rope** [reply]
感谢大佬<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="75" >
